File: abstractos 28-08-2025/abstractos.cpp
#include <iostream>
#include <string>

class Mascota {
public:
    std::string nombre;
    std::string tipo;
    int edad;
    std::string raza;

    Mascota(std::string n, std::string t, int e, std::string r) {
        nombre = n;
        tipo = t;
        edad = e;
        raza = r;
    }
};

int main() {
    Mascota cat("Whiskers", "Gato", 3, "Siames");
    std::cout << "Nombre: " << cat.nombre << ", Tipo: " << cat.tipo << ", Edad: " << cat.edad << ", Raza: " << cat.raza << std::endl;
    return 0;
}

File: abstractos 28-08-2025/abstractos.py
class Mascota:
    def __init__(self, nombre, tipo, edad, raza):
        self.nombre = nombre
        self.tipo = tipo
        self.edad = edad
        self.raza = raza

cat = Mascota("Whiskers", "Gato", 3, "Siames")
print(f"Nombre: {cat.nombre}, Tipo: {cat.tipo}, Edad: {cat.edad}, Raza: {cat.raza}")
